### PR TITLE
Add a `.gitignore.dist` file

### PR DESCRIPTION
**Pull request by ostowe**
This commit will include built files for this plugin in VIP deploys in cases where the plugin is included via submodule.

### CODE REVIEW

File: .gitignore.dist [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Thumbs.db
+phpcs.xml
+phpunit.xml
+wp-cli.local.yml
+node_modules/
+*.sql
+*.tar.gz
+*.zip
+
+# Client build files
```
Comment (benpbolton) [inline .gitignore.dist:12]: Only sanity-test item difference between this and the `built` branch .gitignore is that the build branch also excludes `.npm`, which seems like a fine thing to ignore in both the .gitignore and .gitignore.dist files... might be worth adding.

Comment (benpbolton) [inline .gitignore.dist:12]: I might recommend removing or renaming `Client build files`, as it is no longer excluding 'Client build files' .. in fact, the comment is quite contrary to what it is doing...  /vendor could, for all intents, be listed right alongside node_modules/ or the others if needed...

**Review by benpbolton — CHANGES_REQUESTED** [review]
I think I'd remove (or update) the incorrect comment line at a minimum... totes up to you whether you'd like to add new exclusions to the .gitignore and new .gitignore.dist ... and I wonder if we should start this file with a comment describing what a .gitignore.dist is?

**Comment by ostowe** [comment]
Closing this b/c there's a `built` branch, which I was unaware of when I created it (and serves the same purpose)!

**Pull request closed without merging**